neqo-transport/src/send_stream.rs

```diff
@@ -1320,16 +1642,16 @@ mod tests {
         // Fill the buffer
         assert_eq!(txb.send(&[1; SEND_BUFFER_SIZE * 2]), SEND_BUFFER_SIZE);
         assert!(matches!(txb.next_bytes(),
-			 Some((0, x)) if x.len()==SEND_BUFFER_SIZE
-			 && x.iter().all(|ch| *ch == 1)));
+                         Some((0, x)) if x.len()==SEND_BUFFER_SIZE
```

Comment:
nit: seems like unnecessary changes?

Reply:
replaces tabs with spaces (I did an untabify on the files I changed to avoid introducing tabs, and it already had some).